README.md

```diff
@@ -284,6 +284,52 @@ The solution is to move all of these API calls into background processes, with r
 
 This will improve the user experience, make it fail safe, and improve the development lifecycle.
 
+## Email notifications
+
+We use [GOV.UK Notify](https://notifications.service.gov.uk) to trigger email notifications at various stages for example:
+
+* When a deal status changes in Portal
+* When TFM acknowledges a deal submission
+* When a deal is approved or declined in TFM
+* When a TFM task is ready to start
+
+Each service that triggers an email has it's own "send email" function:
+
+* Portal (BSS): `sendEmail` (currently calls Notify directly)
+* Portal (GEF): `sendEmail` (calls Reference Data Proxy which then calls Notify)
+* TFM: `sendTfmEmail` (calls Reference Data Proxy which then calls Notify)
+
+Each function is very similar - at the moment there is not a way to share between different services. Each function requires:
+
+1) Template ID
+2) Email address
+3) Email variables (object of properties/values to display in the template)
+
+### Notify team members
+
+Only people/email addresses listed in the UKEF Notify 'team members' page, will be able to receive emails and edit templates. 
+
+### How to test emails locally
+
+Currently, all emails are sent to a bank's email address associated with the user(s) that have been involved with the deal creation and submission process.
+
+Therefore, to test eamils we just need to change these emails to your own email that is listed in the Notify team members. To guarantee receiving all possible emails for all services, you should change the emails in the following places:
+
+1) `banks` MongoDB collection > `emails` array. Typically the bank with ID 9 is used.
+2) `users` MongoDB collection > `bank.emails` array. Typically the user `BANK1_MAKER1` is used
+
+If you want to test an already created BSS deal, you'll need to update `deal.details.owningBank.emails`.
```

Comment:
Please add `deal.dealSnapshot.bank.emails` for GEF deals.

Reply:
Great spot but it's not needed 😄 Before TFM submission, GEF doesn't have the bank in the deal, it's only in bank collection. When it's submitted to TFM, TFM grabs the bank from bank collection and adds to the dealSnapshot.

So this part is only relevant for already existing BSS deals

Reply:
Understood, this is good to know. Thanks